.github/workflows/py_ext_main.yml

```diff
@@ -97,12 +97,13 @@ jobs:
       - name: Move wheels and source dist to dist/ directory
         run: |
           mkdir -p dist/
-          mv wheelhouse/*.whl dist/
           mv dist/source-dist/*.tar.gz dist/
+          for wheel_dir in wheelhouse/vttcompilepy*/; do
+            mv "${wheel_dir}"/*.whl dist/
+          done
```

Comment:
Hmm, you know you can do this in one line:
`mv wheelhouse/vttcompilepy*/*.whl dist/` , right?